samples/ibm-samples/openscale/README.md

```diff
@@ -0,0 +1,116 @@
+
+4. A cloud object store
+
+Watson Machine Learning service loads datasets from cloud object store and stores model outputs and other artifacts to cloud object store. Users can use any cloud object store they already preserve. Users can also create a cloud object store with `IBM Cloud Object Storage` service by following this [link](https://console.bluemix.net/catalog/services/cloud-object-storage).
+
+Collect the `endpoint`, `access_key_id` and `secret_access_key` fields from the service credentials for the cloud object store. Create the service credentials first if not exist. To ensure generating HMAC credentials, specify the following in the `Add Inline Configuration Parameters` field: `{"HMAC":true}`.
+
+Create a bucket for storing the train datasets and model source codes.
+
+Then, upload all the files in the `source` folder to the created bucket.
+
+5. Set up access credentials
+
+This pipeline sample reads the credentials from a file hosted in a github repo. Refer to `creds.ini` file and input user's specific credentials. Then upload the file to a github repo the user has access.
+
+To access the credentials file, the user should provide a github access token and the link to the raw content of the file. Modify the `GITHUB_TOKEN` and `CONFIG_FILE_URL` variables in the below code block and run the Python code to create a Kubernetes secret using the KubeFlow pipeline.
+
+```python
+import kfp.dsl as dsl
+import kfp.components as components
+from kfp import compiler
+import kfp
+secret_name = 'aios-creds'
+configuration_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/commons/config/component.yaml')
+client = kfp.Client()
+EXPERIMENT_NAME = 'create secret'
+exp = client.create_experiment(name=EXPERIMENT_NAME)
+@dsl.pipeline(
+    name='create secret',
+    description=''
+)
+def secret_pipeline(
+    GITHUB_TOKEN='',
+    CONFIG_FILE_URL='https://raw.githubusercontent.com/user/repository/branch/creds.ini',
+):
+  get_configuration = configuration_op(
+                  token=GITHUB_TOKEN,
+                  url=CONFIG_FILE_URL,
+                  name=secret_name
+  )
+compiler.Compiler().compile(secret_pipeline,  'secret_pipeline.tar.gz')
+run = client.run_pipeline(exp.id, 'secret_pipeline', 'secret_pipeline.tar.gz')
+```
+
+## Instructions
+
+First, install the necessary Python Packages
+```shell
+pip3 install ai_pipeline_params
+```
+
+In this repository, run the following commands to create the argo files using the Kubeflow pipeline SDK.
+```shell
+dsl-compile --py openscale.py --output openscale.tar.gz
+```
+
+Then, submit `openscale.tar.gz` to the kubeflow pipeline UI. From there you can create different experiments and runs with the OpenScale pipeline.
+
+## Pipeline Parameters
+- **cos-bucket-name**: Object Storage bucket that has the spark training files and openscale manifest
```

Comment:
Object Storage bucket that has Spark training files and OpenScale manifest

Reply:
done